src/main/java/net/minecraftforge/fml/client/GuiSlotModList.java

```diff
@@ -95,19 +95,19 @@ protected void drawSlot(int idx, int right, int top, int height, Tessellator tes
         String       name     = StringUtils.stripControlCodes(mc.getName());
         String       version  = StringUtils.stripControlCodes(mc.getDisplayVersion().isEmpty() ? mc.getVersion() : mc.getDisplayVersion());
         FontRenderer font     = this.parent.getFontRenderer();
-        CheckResult  vercheck = ForgeVersion.getResult(mc);
+        CheckResult  vercheck = ForgeVersion.getCleanResult(mc);
 
         if (Loader.instance().getModState(mc) == ModState.DISABLED)
         {
-            font.drawString(font.trimStringToWidth(name,       listWidth - 10), this.left + 3 , top +  2, 0xFF2222);
-            font.drawString(font.trimStringToWidth(version,    listWidth - (5 + height)), this.left + 3 , top + 12, 0xFF2222);
+            font.drawString(font.trimStringToWidth(name,       listWidth - 10), this.left + 3 , top +  2, 0x993333);
+            font.drawString(font.trimStringToWidth(version,    listWidth - (5 + height)), this.left + 3 , top + 12, 0x993333);
         }
         else
         {
             font.drawString(font.trimStringToWidth(name,    listWidth - 10), this.left + 3 , top +  2, 0xFFFFFF);
             font.drawString(font.trimStringToWidth(version, listWidth - (5 + height)), this.left + 3 , top + 12, 0xCCCCCC);
 
-            if (vercheck.status.shouldDraw())
+            if (vercheck != null && vercheck.status.shouldDraw())
```

Comment:
These colour values can be extracted to static members just so it is easier to keep track.